tutorials/platform/web/javascript_bridge.rst

```diff
@@ -0,0 +1,330 @@
+
+The JavaScriptBridge Singleton
+==============================
+
+In web builds, the :ref:`JavaScriptBridge <class_JavaScriptBridge>` singleton methods
+allows to interact with JavaScript and the browsers and implement some
+functionalities unique to this platform.
+
+Interacting with JavaScript
+---------------------------
+
+Sometimes, when exporting Godot for the Web, it might be necessary to interface
+with external JavaScript code. Things like third-party SDKs, libraries, or
+simply accessing browser features that are not directly exposed by Godot.
+
+The ``JavaScriptBridge`` provides methods to wrap native JavaScript object into
+a special :ref:`JavaScriptObject <class_JavaScriptObject>` that tries to feel
+natural in the context of Godot scripting (e.g. GDScript and C#).
+
+The :ref:`JavaScriptBridge.get_interface() <class_JavaScriptBridge_method_get_interface>`
+meethod retrieves an object in the global scope.
+
+.. code-block:: gdscript
+
+    extends Node
+
+    func _ready():
+        # Retrieve the `window.console` object.
+        var console = JavaScriptBridge.get_interface("console")
+        # Call the `window.console.log()` method.
+        console.log("test")
+
+The :ref:`JavaScriptBridge.create_object() <class_JavaScriptBridge_method_create_object>`
+creates an object via the JavaScript ``new`` constructor.
+
+.. code-block:: gdscript
+
+    extends Node
+
+    func _ready():
+        # Call the JavaScript `new` operator on the `window.Array` object.
+        # Passing 10 as argument to the constructor:
+        # JS: `new Array(10);`
+        var arr = JavaScriptBridge.create_object("Array", 10)
+        # Set the first element of the JavaScript array to the number 42.
+        arr[0] = 42
+        # Call the `pop` function on the JavaScript array.
+        arr.pop()
+        # Print the value of the `length` property of the array (9 after the pop).
+        print(arr.length)
+
+As you can see, by wrapping JavaScript objects into ``JavaScriptObject`` you can
+interact with them like they were native Godot objects, calling their methods,
+and retrieving (or even setting) their properties.
+
+Base types (int, floats, strings, booleans) are automatically converted (floats
+might lose precision when converted from Godot to JavaScript). Anything else
+(i.e. objects, arrays, functions) are seen as ``JavaScriptObjects`` themselves.
+
+Callbacks
+---------
+
+Calling JavaScript code from Godot is nice, but sometimes you need to call a
+Godot function from JavaScript instead.
+
+This case is a bit more complicated. JavaScript relies on garbage collection,
+while Godot uses reference counting for memory management. This means you have
+to explicitly create callbacks (which are returned as ``JavaScriptObjects``
+themselves) and you have to keep their reference.
+
+Arguments passed by JavaScript to the callback will be passed as a single Godot
+``Array``.
+
+But don't worry if this sounds a bit technical, it's not too hard.
```

Comment:
I thought we avoided saying things like simple, easy etc, not reassuring people of things not being "not too hard".

I feel like removing this line will make people freak out because they don't understand the terms above, and just give up.